2/contrib/openshift/base-plugins.txt

```diff
@@ -42,12 +42,12 @@ branch-api:1.10
 
 # Pipeline SCM Step Plugin - https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+SCM+Step+Plugin
 workflow-scm-step:2.1
-git:2.4.4
+git:3.0.0
 mapdb-api:1.0.1.0
 subversion:2.5.7
 
 # mercurial - https://wiki.jenkins-ci.org/display/JENKINS/Mercurial+Plugin
-mercurial:1.56
+mercurial:1.54
```

Comment:
yeah this is intentional, i had bumped out of sync w/ v1 during some v2 testing but i don't believe it was necessary, so i'm re-aligning them.